ci/test/00_setup_env_win32.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2019-2020 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+export LC_ALL=C.UTF-8
+
+export CONTAINER_NAME=ci_win32
+export DOCKER_NAME_TAG=ubuntu:18.04  # Check that bionic can cross-compile to win32 (bionic is used in the gitian build as well)
+export HOST=i686-w64-mingw32
+export PACKAGES="python3 nsis g++-mingw-w64-i686 wine-binfmt wine32"
+export RUN_UNIT_TESTS=false
+export RUN_FUNCTIONAL_TESTS=false
```

Comment:
We currently don't include win-binfmt or wine32/64, and I would not think it is necessary unless you are going to run the unit tests for the windows executable under WINE.

Reply:
Also turned this off after `test/fs_tests.cpp` error. Forgot to remove the dependencies, will do.